### ISSUE TITLE
There should be add a sample/ Demo option available for Users to view before downloading a huge file

### ISSUE THREAD

**Issue by gptshubham595** [kiwix/kiwix-android#1087]
**There should a option given that on tapping on any content it should provide two option 1) See Sample 2)Download 
![to](https://user-images.githubusercontent.com/24877361/54489454-594c1080-48d2-11e9-85c0-ee0a907a7c43.png)**
![to1](https://user-images.githubusercontent.com/24877361/54489469-90222680-48d2-11e9-8e37-6e0c3fddeb1d.png)

*And Sample View should display one sample page how that downlaoded content should look*

**Comment by mhutti1** [comment]
I don't really think this is something we can easily implement in Android. Perhaps link to the real website if its online?

**Comment by gptshubham595** [comment]
No, when the user is gonna download that he will be under internet service so before downloading he can click and go through the website

**Comment by kelson42** [comment]
Moving to `kiwix/libkiwix` because at the end the library.kiwix.org ODPS API should provide al ink for that first. Once this is done, we could think of udating the readers.
